source/mesh_system.c

```diff
@@ -64,17 +64,20 @@ static void mesh_system_heap_error_handler(heap_fail_t event)
 void mesh_system_init(void)
 {
     if (mesh_initialized == false) {
+#ifndef YOTTA_CFG
+        ns_hal_init(app_stack_heap, MESH_HEAP_SIZE,
+                    mesh_system_heap_error_handler, NULL);
+        eventOS_scheduler_mutex_wait();
+        net_init_core();
+        eventOS_scheduler_mutex_release();
```

Comment:
The existing "central registration" SAL is not present, so there's no equivalent initialisation. This stack startup is called by a "standalone" socket/interface adaptation.